public/assets/js/dashboard.js

```
@@ -0,0 +1,200 @@
+    const numeroRueFields = document.getElementById('numero_rue_fields');
+    const lieuDitField = document.getElementById('lieu_dit_field');
+
+    function toggleAdresseFields() {
+        const selectedType = document.querySelector('input[name="adresse_type"]:checked').value;
+        
+        if (selectedType === 'numero_rue') {
+            numeroRueFields.style.display = 'block';
+            lieuDitField.style.display = 'none';
+            // Activer les champs numéro et rue
+            numeroRueFields.querySelectorAll('input').forEach(input => input.required = true);
+            lieuDitField.querySelector('input').required = false;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)